draft-lmbdhk-quic-multipath.md

```diff
@@ -517,45 +518,111 @@ not cause linkability issue.
 ~~~
 {: #fig-example-new-path title="Example of new path establishment"}
 
-As shown in {{fig-example-new-path}}, client provides one unused available Connection ID (C1 with sequence number 1),
-and server provides two available Connection IDs (S1 with sequence number 1, and S2 with sequence number 2).
-When client wants to start a new path, it checks whether there is unused available Connection IDs for each side,
-and choose an available Connection ID S2 as the Destination Connection ID in the new path.
+As shown in {{fig-example-new-path}}, client provides one unused available
+Connection ID (C1 with sequence number 1), and server provides two available
+Connection IDs (S1 with sequence number 1, and S2 with sequence number 2).
+When client wants to start a new path, it checks whether there is unused
+available Connection IDs for each side, and choose an available Connection ID
+S2 as the Destination Connection ID in the new path.
 
-Endpoints need to exchange unused available Connection IDs with the NEW_CONNECTION_ID frame before an endpoint
-starts a new path. For example, if the goal is to maintain 2 paths, each endpoint should provide at least 3 CID to its peer:
-2 in use, and one spare. If the client has used all the allocated CID, it is supposed to retire those that are
-not used anymore, and the server is supposed to provide replacements, as specified in {{QUIC-TRANSPORT}}.
+Endpoints need to exchange unused available Connection IDs with the
+NEW_CONNECTION_ID frame before an endpoint starts a new path. For example, if
+the goal is to maintain 2 paths, each endpoint should provide at least 3 CID to
+its peer: 2 in use, and one spare. If the client has used all the allocated
+CID, it is supposed to retire those that are not used anymore, and the server
+is supposed to provide replacements, as specified in {{QUIC-TRANSPORT}}.
 
 
 ## Path Closure
 
-{{fig-example-path-close}} illustrates an example of path closing. In this
-case, we are going to close the first path. For the first path, the server's
-1-RTT packets use DCID C1, which has a sequence number of 1; the client's 1-RTT
-packets use DCID S2, which has a sequence number of 2.  For the second path,
-the server's 1-RTT packets use DCID C2, which has a sequence number of 2; the
-client's 1-RTT packets use CID S3, which has a sequence number of 3. Note that
-two paths use different packet number space.
+{{fig-example-path-close1}} illustrates an example of path closing when both
+the client and the server use non-zero length CIDs. Path identifier type 0x00
+is chosen. In this case, we are going to close the first path. For the first
```

Comment:
I would say "... use non-zero length CIDs. For the first path, the server's 1-RTT packets use DCID C1... Note that two paths use different packet number spaces." Then, I would complement by "In this case, the client is going to close the first path. It identifies it by the sequence number of the DCID it uses over that path (path identifier type 0x00), hence using the path_id 1. In reply, the server confirms the path using the sequence number of the DCID it uses over that path (path identifier type 0x00), corresponding to path_id 2.